dashboard/db/migrate/20200928224602_create_join_table_lesson_resource.rb

```diff
@@ -0,0 +1,8 @@
+class CreateJoinTableLessonResource < ActiveRecord::Migration[5.0]
+  def change
+    create_join_table :lessons, :resources do |t|
+      # t.index [:lesson_id, :resource_id]
+      # t.index [:resource_id, :lesson_id]
```

Comment:
We will definitely want indexes on lesson_id and resource_id. I would suggest accomplishing that by uncommenting this code, since that is what I see for other join tables.